ledger-core/virtual/tables/objects_request_table.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package tables
```

Comment:
please, try to avoid naming of package in plural form

Reply:
also i'd be glad if it can give more information - what table? table is pretty common name for data structure

Reply:
Why we use
```
type WorkingTable struct {
	lists map[contract.InterferenceFlag]*WorkingList
}
```
 what does table mean in this case? Same comment for this structure.

I may change the package name to 'store'. It will be more clear?

Reply:
You can ask @cyraxred in that case, I don't know good/clear name for that package. 

`tables` is not a good name, anyway.

Reply:
Fixed, rename the package to 'callregistry'